plugin/ui/chat.py

```diff
@@ -117,8 +126,11 @@ def conversation(self, value: list[CopilotPayloadConversationEntry]) -> None:
     # normal methods #
     # -------------- #
 
-    def __init__(self, window: sublime.Window) -> None:
-        self.window = window
+    def __init__(self, window: sublime.Window | None) -> None:
+        if not window:
+            raise ValueError("window is required")
+
+        self.window: sublime.Window = window
```

Comment:
What's going on? If we need this check, then the issue is probably in other places.

Reply:
Ci-check was complaining